cairo/ethereum/cancun/trie.cairo

```diff
@@ -332,10 +344,30 @@ func encode_node{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: Kecc
 //         // trie._data[key] = value
 // }
 
-// func trie_get(trie: Trie[K, V], key: K) -> V {
-//     // Implementation:
-//     // return trie._data.get(key, trie.default)
-// }
+func trie_get_TrieAddressAccount(trie: TrieAddressAccount, key: Address) -> Account {
+    let dict_ptr = cast(trie.value._data.value.dict_ptr, DictAccess*);
+
+    with dict_ptr {
+        let (pointer) = dict_read(key.value);
+    }
+    let new_dict_ptr = cast(dict_ptr, AddressAccountDictAccess*);
```

Comment:
What the use of this ?

Reply:
probably missing an update to the trie.value._data.value

Reply:
yes